recipe/meta.yaml

```
@@ -23,42 +20,42 @@ requirements:
     - setuptools
     - pip
     - wheel
+    - flit-core <4
 
   run:
     - python
-    - nltk >=3.1
+    - nltk >=3.8
 
-{% set tests_to_ignore = "--ignore=tests/tests/test_blob.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_classifiers.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_np_extractor.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_tokenizers.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_sentiments.py" %}
-{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/tests/test_taggers.py" %}
 
 test:
   source_files:
     - tests
+    - pyproject.toml
   imports:
     - textblob
     - textblob.en
     - textblob.unicodecsv
   commands:
     - pip check
-    - pytest -v tests {{ tests_to_ignore }}
+    # Downloading nltk assets for testing. textblob.download corpora is insufficient for testing needs.
+    - python -c "import nltk; nltk.download('punkt_tab')"
+    - python -c "import nltk; nltk.download('averaged_perceptron_tagger_eng')"
+    - python -m textblob.download_corpora
+    - pytest -v tests
```

```suggestion
    - pytest -v tests -m "not slow"
```
I would skip all tests that are marked as slow.

---

There is a 6 second delta between all tests and not slow tests but I will go ahead and leave them out.

---

There might be a 6 second now, but it could be significantly bigger than that in the future... So it's a little bit to save resources now and also save us from accidentally not seeing that the build time increased int the future.